wpengine/faustjs
# [Merl 1256] Bug fix. Generate Autosave is missing from Post/Page Previews

## Tasks


## Description



Fixes missing call to `autosave` when clicking on the Preview links.

## Related Issue(s):



## Testing



1. Checkout this branch that includes changes to the faustwp plugin
2. Using the updated faustwp plugin, create a new post & save as draft (do not publish)
3. Observe that both preview links correctly resolve to display draft content.
4. Verify that the draft content is also presented in the headless site.

## Screenshots



## Documentation Changes



## Dependant PRs
